proposed/2022/NuGetExeWarnWhenSDKProjectsGetPacked.md

```diff
@@ -0,0 +1,97 @@
+# Show error when using NuGet.exe to pack an SDK csproj

```

Comment:
Can we stop using the `proposed` folder, please?  This PR is the proposal, and if it gets merged then it shows the proposal is accepted.  Given we're now using pull requests, not the wiki, for designed, I see no reason to have a `proposed` folder any longer.

Reply:
The only reason we have a proposed folder is that things might not be accepted to be worked on even if a PR is ready to merge said design. We can change that and just use PRs though. I just based the initial flow off other popular RFC processes.